pymeasure/instruments/hp/hp437b.py

```diff
@@ -0,0 +1,949 @@
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import strict_discrete_set, strict_range
+from enum import Enum, IntFlag
+
+import logging
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class MeasurementUnit(Enum):
```

Comment:
Did you consider using an `IntEnum`?

I have not yet seen your code, but an IntEnum behaves like an `int`, which might become handy.

Reply:
The question probably boils down to if the user shall be able to specify a integer for the measurement unit. What do you think ?

Reply:
What I meant is, that you simplify checks in your code.

For example, below you do `e: int(f.value)`, but instead, you can reduce it (for an IntEnum) to `e: e for e in MeasurementUnit` as the comparison will be an integer comparison automatically.

For the user, nothing changes.